bioc/eset_sumexp.Rmd

```diff
@@ -14,29 +14,33 @@ library(GEOquery)
 ```
 
 ```{r}
-geoq <- getGEO("GSE9514")
+geoq <- getGEO("GSE9514")    # download a dataset from GEO
 names(geoq)
-e <- geoq[[1]]
+e <- geoq[[1]]    # extract ExpressionSet
 ```
 
 ### ExpressionSet
 
 ```{r}
-dim(e)
+# exprs gives expression matrix
+dim(e)    # number of features and samples in ExpressionSet
 exprs(e)[1:3,1:3]
```

Comment:
have you tried `exprs(e[1:3,1:3])` ... it is better practice because `exprs(e)[1:3,1:3]` must compute the whole matrix `exprs(e)` ...

Reply:
Are you referring to line 27, which was unchanged in my previous commit? I am happy to change that if you want, but that is the format in the textbook and how it is shown in the course. Having said that, there are many places in the Rmds where this would need to be changed if we choose to do that.

Would you like that done?